### DISCUSSION TITLE
Provide LDBC benchmark for SF 1000

### DISCUSSION THREAD

**Discussion by whitewum** [alibaba/GraphScope#13] [Show and tell]
**Is your feature request related to a problem? Please describe.**

I saw that the [LDBC benchmark data set](https://github.com/alibaba/GraphScope/tree/main/interactive_engine/benchmark)  is small and  generator.scaleFactor = 30.  

According to LDBC snb datagen, the largest data set can be SF1000 (and 3000+ WIP)

**Describe the solution you'd like**

Can you provide the benchmark results on SF1000?

**Describe alternatives you've considered**

**Additional context**

**Comment by zhengpingq** [comment]
Hi, @whitewum 

Thanks a lot for raising this issue!

We have only included the results using `SF30` so that our benchmark results are easy to reproduce--it is faster to load a relatively small dataset.

Having said that, we have done extensive evaluation<sup>1</sup> for the interactive engine in GraphScope, reported in Section 6 of this paper ([preprint](https://www.researchgate.net/profile/Zhengping_Qian/publication/347910256_GraphScope_A_System_for_Interactive_Analysis_on_Distributed_Graphs_Using_a_High-Level_Language/links/5fe6efff92851c13febdcbf7/GraphScope-A-System-for-Interactive-Analysis-on-Distributed-Graphs-Using-a-High-Level-Language.pdf)), which includes performance analysis in terms of query latency and scalability, as well as microbenchmarks for key system mechanisms supporting parallel graph-traversal computation.

Please let us know if further information is needed. Thank you!

<sup>1</sup>. The tests have been performed using internal development branch, which will be released as open source in the coming year.
